src/edu/stanford/nlp/mt/util/ParallelSuffixArray.java

```diff
@@ -39,9 +36,9 @@
   private static final Logger logger = LogManager.getLogger(ParallelSuffixArray.class);
   
   protected int[] srcBitext;
-  protected int[] f2e;
+  protected Set<Integer>[] f2e;
```

Comment:
This change (and the change to `e2f` 2 lines further down) seems to be the source of the dimension change elsewhere in the API.  Why are we going from an array of integers to an array of collections of integers?